shared/actions/config/index.tsx

```diff
@@ -465,23 +462,25 @@ function* criticalOutOfDateCheck() {
   // check every hour
   while (true) {
     try {
-      const s = yield* Saga.callPromise(RPCTypes.configGetUpdateInfo2RpcPromise, {})
+      const s: Unpacked<ReturnType<typeof RPCTypes.configGetUpdateInfo2RpcPromise>> = yield* Saga.callPromise(
```

Comment:
Can we get the typing from `callPromise`?

Reply:
not really. callPromise returns an effect. The effect is processed by the saga (which gets it from the yield) and returns the value of the promise. For this type to work automatically tsc would have to understand that interplay (which isn't feasible right now)